src/modules/logger/logger.cpp

```diff
@@ -624,6 +624,14 @@ void Logger::add_thermal_calibration_topics()
 	add_topic("sensor_gyro", 100);
 }
 
+void Logger::add_sensor_comparison_topics()
+{
+	add_topic("sensor_accel", 10);
```

Comment:
Just to clarify: 10 is the interval in ms, not Hz. Is that your intention?

Reply:
No it wasn't, i will change it to 100.